manifests/php.yml

```diff
@@ -455,9 +455,9 @@ tests/:
     test_128_bit_traceids.py:
       Test_128_Bit_Traceids: v0.84.0
     test_config_consistency.py:
-      Test_Config_Dogstatsd: missing_feature
+      Test_Config_Dogstatsd: v1.9.0+
```

Comment:
Did you mean simply v1.9.0, without the `+`?

Reply:
I added the `+` so it runs against the changes in the main branch

Reply:
The `dev` branch looks at dd-trace-php/master, if it's equal to the upcoming version.

Reply:
Yeah, the artifact produced by CI on master is the next minor compared to what's contained in the VERSION file, plus a git sha hash. So that works nicely for PHP :-)